handle '.' in package.json correctly

Currently npm2deb charm fails because of this section in package.json

```
"directories" : {
        "lib" : ".",
        "example" : "example",
        "test" : "test"
    },
```

This is the error when trying to build,

```
cp: cannot copy a directory, './.', into itself, 'debian/node-charm/usr/lib/nodejs/charm/.'
dh_install: cp --reflink=auto -a ./. debian/node-charm/usr/lib/nodejs/charm// returned exit code 1
debian/rules:8: recipe for target 'binary' failed
```

--- a/npm2deb/__init__.py
+++ b/npm2deb/__init__.py
@@ -267,6 +267,11 @@ and may not include tests.\n""")
             else:
                 libs.add('*.js')
 
+        # normalize filepaths
+        libs = set(map(lambda x: _os.path.normpath(x), libs))
+        # sanitize by removing current directory from files list
+        libs.remove('.')
+
         for filename in libs:
             content += "%s %s/\n" % (filename, self.debian_dest)
         utils.create_debian_file('install', content)

